### ISSUE TITLE
Install sl before using it

### ISSUE THREAD

**Issue by ghost** [0xblank/.croissantrc#2]
Maybe you could install sl packet before using it in croissantrc ?

**Comment by 0xblank** [comment]
I though about that but the is that it would require to have an open root shell or a user shell with sudo password in cache which is to me quite unlikely. But I could update the script to check for the current user and if root install the package. I'll do it if I have time. Thanks for the issue, don't hesitate to star the project if you like it :)

**Comment by ghost** [comment]
In this case, maybe could you do another thing if sl isn't installed yet ? For example "ls > /dev/null" to display nothing, like an empty folder

**Comment by 0xblank** [comment]
yes I could do that, I'll see if I have time to change this. I you want to do a pull request don't hesitate :)
